From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Davis Goodin <dagood@microsoft.com>
Date: Fri, 15 Oct 2021 02:41:32 -0500
Subject: [PATCH] Exclude src/Dependencies from source-build

This makes source-build stop producing the Microsoft.CodeAnalysis.Collections package. Downstream repos (MSBuild) can't safely use a source-built version.

PR for applying this patch: https://github.com/dotnet/roslyn/pull/57165
---
 .../{CodeAnalysis.Debugging => }/Directory.Build.props      | 0
 src/Dependencies/Microsoft.NetFX20/Directory.Build.props    | 6 ------
 src/Dependencies/PooledObjects/Directory.Build.props        | 6 ------
 3 files changed, 12 deletions(-)
 rename src/Dependencies/{CodeAnalysis.Debugging => }/Directory.Build.props (100%)
 delete mode 100644 src/Dependencies/Microsoft.NetFX20/Directory.Build.props
 delete mode 100644 src/Dependencies/PooledObjects/Directory.Build.props

diff --git a/src/Dependencies/CodeAnalysis.Debugging/Directory.Build.props b/src/Dependencies/Directory.Build.props
similarity index 100%
rename from src/Dependencies/CodeAnalysis.Debugging/Directory.Build.props
rename to src/Dependencies/Directory.Build.props
diff --git a/src/Dependencies/Microsoft.NetFX20/Directory.Build.props b/src/Dependencies/Microsoft.NetFX20/Directory.Build.props
deleted file mode 100644
index 6eef643958f..00000000000
--- a/src/Dependencies/Microsoft.NetFX20/Directory.Build.props
+++ /dev/null
@@ -1,6 +0,0 @@
-<Project>
-  <Import Project="$([MSBuild]::GetPathOfFileAbove('Directory.Build.props', '$(MSBuildThisFileDirectory)../'))" />
-  <PropertyGroup>
-    <ExcludeFromSourceBuild>true</ExcludeFromSourceBuild>
-  </PropertyGroup>
-</Project>
diff --git a/src/Dependencies/PooledObjects/Directory.Build.props b/src/Dependencies/PooledObjects/Directory.Build.props
deleted file mode 100644
index 6eef643958f..00000000000
--- a/src/Dependencies/PooledObjects/Directory.Build.props
+++ /dev/null
@@ -1,6 +0,0 @@
-<Project>
-  <Import Project="$([MSBuild]::GetPathOfFileAbove('Directory.Build.props', '$(MSBuildThisFileDirectory)../'))" />
-  <PropertyGroup>
-    <ExcludeFromSourceBuild>true</ExcludeFromSourceBuild>
-  </PropertyGroup>
-</Project>
